src/main.ts

```diff
@@ -295,7 +313,8 @@ export class ConnectionFeature extends SqlOpsFeature<undefined> {
 			rebuildIntelliSenseCache,
 			registerOnConnectionChanged,
 			registerOnIntelliSenseCacheComplete,
-			registerOnConnectionComplete
+			registerOnConnectionComplete,
+			changePassword
```

Comment:
nit: it looks like the methods are arranged with "normal" methods first and registration methods at the end.  Probably when updating it makes sense to following the existing convention in the code, e.g., adding this method on line 310 after the other "change" method.